Userland/Libraries/LibGUI/Button.cpp

```diff
@@ -132,11 +132,11 @@ void Button::set_action(Action& action)
         set_checked(action.is_checked());
 }
 
-void Button::set_icon(RefPtr<Gfx::Bitmap>&& icon)
+void Button::set_icon(RefPtr<Gfx::Bitmap> icon)
 {
     if (m_icon == icon)
         return;
-    m_icon = move(icon);
+    m_icon = icon;
```

Comment:
If you take the `RefPtr` by value, there's absolutely no downside to ´move()´ing from it, since you'll only `move()` from the value that has been copied onto your call frame.